js/map.js

```diff
@@ -89,7 +89,7 @@ window.map = (function () {
     if (dragged) {
       mapPinsMainElement.addEventListener('click', clickPreventDefaultHandler);
     }
-    window.debounce(applyFilter);
+    renderPins(data);
```

Comment:
you need to start loading data when the page is active and render data in the callback of the request function.
You still have a bug here because data is loaded asynchronously.

Reply:
render data inside of the receiveDataHandle

Reply:
done